miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -347,7 +348,7 @@ var HotUtils = {
       for (var i = 0; i < data.length; i++) {
         var flat = flatObjects[i];
         var obj = data[i];
-        column.update(obj, flat, flat[column.depends], function(readOnly) {
+        column.update(obj, flat, '*start', flat[column.depends], function(readOnly) {
           table.setCellMeta(i, column.hotIndex, 'readOnly', readOnly);
           needsRender = true;
         }, function(optionsObj) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define this function outside of a loop. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AFunctionDefinitionInsideLoop)